packages/docusaurus/src/webpack/minification.ts

```diff
@@ -86,17 +86,21 @@ function getAdvancedCssMinifier() {
 }
 
 export function getMinimizer(): WebpackPluginInstance[] {
+  const minimizer: WebpackPluginInstance[] = [
+    new rspack.SwcJsMinimizerRspackPlugin(),
+    new rspack.SwcCssMinimizerRspackPlugin(),
```

Comment:
use https://rspack.dev/plugins/rspack/lightning-css-minimizer-rspack-plugin instead, SwcCssMinimizerRspackPlugin is removed infavor of lightning-css-minimizer-rspack-plugin